pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -292,18 +322,56 @@ process runIqtree {
     """
 }
 
+// Function to check if a genome is present in
+// the annotation cache.
+def annoInCache(genome, anno_cache) {
```

Comment:
Does the annotation cache need to handle updated input genomes? For example, if for some reason a genome FASTA file is updated, is it forwarded to `linkAnnoCache` or `buscoAnnot`?

Reply:
No, the name of the fasta has to be changed after update or deleted from the cache to trigger reannotation.

Reply:
OK, thanks. I was trying to get a sense of what the cache is expected to do.